wifiphisher/pywifiphisher.py

```diff
@@ -187,7 +187,8 @@ def shutdown(template=None, network_manager=None):
         template.remove_extra_files()
 
     print '[' + R + '!' + W + '] Closing'
-    sys.exit(0)
+    for c in phishinghttp.creds:
+        sys.exit('[' + R + '+' + W + '] Captured credentials:' + R + c)
```

Comment:
In a little more detail:

Log all jamming interfaces -> append to list (1)

Log all DHCP leases -> append to list (2)

Log all HTTP requests -> append to list (3)

Each time one of the list gets updated, save this to a log stream file, then tail the entire log stream.